### ISSUE TITLE
Balloon is no longer moveable (keyboard)

### ISSUE THREAD

**Issue by terracoda** [phetsims/balloons-and-static-electricity#462]
A generalizable change to GFL's ruler's help text was made to BASE and Friction. See https://github.com/phetsims/gravity-force-lab/issues/216#issuecomment-570003090

While verifying the content change in BASE, I noticed the Balloon was not draggable with the keyboard. WASD and Arrow keys do not work when balloon is grabbed. I can jump the balloon, though with special jump keys (JW, JS, and JC).

I have NOT tested the interaction on mobile.

I am not sure when the balloon stopped moving. The changes that @zepumph made were likely only content related. The book in Friction and the ruler in GFL are still draggable with the keyboard and work fine.

Assigning to @zepumph and @jessegreenberg. Please adjust assignment as needed.

**Comment by zepumph** [comment]
In 41e3884f5e5b89a05491657bf39d791100acd0fc "locationProperty" was changed to "positionProperty" for a KeyboardDragListener listener as part of https://github.com/phetsims/phet-info/issues/126, but the KeyboardDragListener api was not changed until I just committed. This should be fixed now, tested on master. Closing
